### PR TITLE
Fix `/web` prefix

### PR DESCRIPTION
**Pull request by ykzts**

### CODE REVIEW

**Comment by ClearlyClaire** [comment]
Fixes the 500 error, but old paths result in client-side 404, while it should probably redirect in order to not break history.

**Comment by Gargron** [comment]
The weird thing is that on localhost, old paths *do* redirect me to the new ones.

File: config/routes.rb [hunk 1]
```diff
@@ -681,9 +681,10 @@
     get path, to: 'home#index'
   end
 
-  get '/web/(*any)',   to: redirect('/%{any}', status: 302), as: :web
```
Comment (Gargron) [inline config/routes.rb:684]: I think `defaults: { any: '' }` can be used to fix this

**Comment by ClearlyClaire** [comment]
It does work… when the service worker is not registered. Otherwise it caches it.

**Review by Gargron — APPROVED** [review]

**Pull request merged**